src/NativeShell/www/apphost.js

```diff
@@ -0,0 +1,172 @@
+//disables all console logs
+console.log = function () {};
```

Comment:
can we remove this?

Reply:
I added to maintain the original functionality. I can remove this and all console logs made in jf-web will be outputted to the console.

Reply:
I decided to remove, now one can see what outputs in console when working in real time for debug purposes.